src/components/Calculator.js

```diff
@@ -1,42 +1,67 @@
+  handleClick(event) {
+    const value = event.target.textContent;
+    const solution = calculate(this.state, value);
+    this.setState(solution);
+  }
+
+  /* eslint-disable react/destructuring-assignment */
   render() {
+    const { total, next, operation } = this.state;
     return (
       <div className="calculatorMain">
-        <p className="display">this is a test 006  </p>
-        <KeyBoard />
+        <p className="display">
+          { total }
+          { operation }
+          { next }
+        </p>
+        <KeyBoard handler={this.handleClick} />
       </div>
 
     );
   }
 }
 
-const KeyBoard = () => {
+const KeyBoard = (props) => {
   return (
     <div className="KeyBoard">
-      <button type="button" className="btnPad0">0</button>
-      <button type="button" className="btnPad1">1</button>
-      <button type="button" className="btnPad2">2</button>
-      <button type="button" className="btnPad3">3</button>
-      <button type="button" className="btnPad4">4</button>
-      <button type="button" className="btnPad5">5</button>
-      <button type="button" className="btnPad6">6</button>
-      <button type="button" className="btnPad7">7</button>
-      <button type="button" className="btnPad8">8</button>
-      <button type="button" className="btnPad9">9</button>
-      <button type="button" className="btnPadAdd">+</button>
-      <button type="button" className="btnPadMinus">-</button>
-      <button type="button" className="btnPadDiv">/</button>
-      <button type="button" className="btnPadMultiply">X</button>
-      <button type="button" className="btnPadEqual">=</button>
-      <button type="button" className="btnPadPercentage">%</button>
-      <button type="button" className="btnPadAllClear">AC</button>
-      <button type="button" className="btnPadMasMin">+/-</button>
-      <button type="button" className="btnPadDecimal">.</button>
+      <button type="button" onClick={props.handler} className="btnPad0">0</button>
+      <button type="button" onClick={props.handler} className="btnPad1">1</button>
+      <button type="button" onClick={props.handler} className="btnPad2">2</button>
+      <button type="button" onClick={props.handler} className="btnPad3">3</button>
+      <button type="button" onClick={props.handler} className="btnPad4">4</button>
+      <button type="button" onClick={props.handler} className="btnPad5">5</button>
+      <button type="button" onClick={props.handler} className="btnPad6">6</button>
+      <button type="button" onClick={props.handler} className="btnPad7">7</button>
+      <button type="button" onClick={props.handler} className="btnPad8">8</button>
+      <button type="button" onClick={props.handler} className="btnPad9">9</button>
+      <button type="button" onClick={props.handler} className="btnPadAdd">+</button>
+      <button type="button" onClick={props.handler} className="btnPadMinus">-</button>
+      <button type="button" onClick={props.handler} className="btnPadDiv">+</button>
```

Comment:
- [ ] This button suppose to be for division calculation `÷` not addition `+` so please fix that to make the calculator work as expected.

![image](https://user-images.githubusercontent.com/68796232/188988739-7c05ff5d-a5b1-4da8-a607-80dbafbbe69e.png)